checker/jtreg/nullness/defaultsPersist25/Classes25.java

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
Can you add a README.md file in each directory that explains the relation between the different files? Can you cross-reference between the old and the new versions, to make this relationship discoverable?